software/pretix.yml

```diff
@@ -5,6 +5,8 @@ licenses:
   - Apache-2.0
 platforms:
   - Python
+  - Docker
 tags:
   - E-commerce
-source_code_url: https://github.com/pretix
+  - Polls and Events
```

Comment:
```suggestion
```

The previous category `E-commerce` was fine. I don't think it fits very well in `Polls and events` since it's a sales/e-shop platform and `Polls and events` is for the organization/scheduling of events (see https://awesome-selfhosted.net/tags/polls-and-events.html)